Directory.Build.targets

```diff
@@ -1,10 +1,17 @@
 <Project>
 
   <Import Project="$(RepositoryEngineeringDir)resources.targets" />
-  <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true' and '$(IsSourceProjectPackable)' == 'true'">
+    <!-- IsSourceProjectPackable is used in the CI packing leg to enable IsPackable only on source projects.
+         This needs to be evaluated before importing the Arcade SDK and packaging.targets. -->
+    <IsPackable>true</IsPackable>
```

Comment:
This is set too late, properties in the project body using `$(IsPackable)` will not see this value.  Need to do this in props.